core/src/main/java/com/dtolabs/rundeck/core/execution/impl/jsch/JschNodeExecutor.java

```diff
@@ -165,6 +165,10 @@
     public static final String FWK_SSH_CONFIG_PREFIX = FWK_PROP_PREFIX + SSH_CONFIG_PREFIX;
     public static final String PROJ_SSH_CONFIG_PREFIX = PROJ_PROP_PREFIX + SSH_CONFIG_PREFIX;
 
+    public static final String NODE_ATTR_PASS_ENV = "ssh-accept-env";
```

Comment:
suggest changing to ssh-send-env
```suggestion
    public static final String NODE_ATTR_PASS_ENV = "ssh-send-env";
```